internal/pkg/util/fs/helper.go

```diff
@@ -176,8 +176,8 @@ func MakeTmpFile(basedir, pattern string, mode os.FileMode) (*os.File, error) {
 	return f, nil
 }
 
-// FileExists simply checks if a path exists.
-func FileExists(path string) (bool, error) {
+// PathExists simply checks if a path exists.
+func PathExists(path string) (bool, error) {
```

Comment:
Minor point, I would keep `FileExists`, add `PathExists` and update `FileExists` to check whether the path is a file. Feel free to ignore.

Reply:
I was thinking this, but this helper package already has `IsFile()` which covers the use case for `FileExists()` for the most part